### DISCUSSION TITLE
Where is Geonode base html template?

### DISCUSSION THREAD

**Discussion by Raji-Ahmad** [GeoNode/geonode#11732] [Q&A] [answered]
Where is Geonode base HTML template? I need to add some traffic analytics to my website and I've got to edit the base template to effect the change in all pages. 

I tried editing "geonode/geonode/templates/base.html", but no effect. 

I can't fine index.html either because "geonode/geonode/templates/index.html says "{# index.html is implemented by the external client #}".

**Comment by nicokant** [comment, marked as answer]
Hi, I think you can find it in geonode-mapstore-client: https://github.com/GeoNode/geonode-mapstore-client/tree/master/geonode_mapstore_client/templates
